v2/calendar.go

```diff
@@ -0,0 +1,17 @@
+package moviebuff
```

Comment:
you could import parent package and only implement the new functions or you could move the files here and import v2 package in parent package then keep only the old funcs in the parent package.

Reply:
Yeah. There should not need to declare these structs again. Think of v2 as a wrapper on top of current methods. Let's discuss if you have any doubts about how to approach this @thamizhv.

Reply:
@sivachandran  As discussed, added parent dependency in v2.